### ISSUE TITLE
LSTM Model Prototype

### ISSUE THREAD

**Issue by sjawhar** [sjawhar/no-wander#2]
## Expected Outcome
One of the classifiers worth trying is LSTM;
* Ingest epochs
* Split and label pre/post recovery
* Split into 128-reading samples
* Featurization
  ![Features](https://user-images.githubusercontent.com/5074378/70933126-1c0bdf00-201a-11ea-8772-437bb0a7fd0d.png)
  * Along with DFT, the Discrete Wavelet Transform is also applied using a 7-level decomposition and the Daubechies 4 (db4) as the mother wavelet, to extract the detail (64–128 Hz, 32–64 Hz, 16–32 Hz, 8–16 Hz, 4–8 Hz, 2–4 Hz, 1–2 Hz) and approximation coefficients (<1 Hz). The 7-level decomposition is selected based on the 256 Hz sampling frequency used when assembling the CHB-MIT database, as it is the minimum depth required to cover the fundamental frequency bands and allow for the separation of bellow the 1 Hz that is most predominantly occupied by artifacts.
  * Phase locking
  * Spike rate
  * Feature extractor CNN?
* Train/test split
* Tweak those parameters, yo!

## Resources
- [x] [Emotion Recognition based on EEG using LSTM](https://thesai.org/Downloads/Volume8No10/Paper_46-Emotion_Recognition_based_on_EEG_using_LSTM.pdf)
- [x] [A Long Short-Term Memory deep learning network for the prediction of epileptic seizures using EEG signals](https://www.sciencedirect.com/science/article/pii/S001048251830132X)
- [x] [Deep long short-term memory structures model temporal dependencies improving cognitive workload estimation](https://www.sciencedirect.com/science/article/pii/S0167865517301678)
- [ ] [Sequence Classification with LSTM Recurrent Neural Networks in Python with Keras](https://machinelearningmastery.com/sequence-classification-lstm-recurrent-neural-networks-python-keras/)
